v2/brigadier/src/projects.ts

```diff
@@ -1,4 +1,9 @@
+/**
+ * A Brigade project.
+ */
```

Comment:
Nit: I don't know if it's misleading at all to just call this "a Brigade project." The reality is that Brigadier / the worker see only a relevant subset of all event and project details. i.e. It's a Brigadier / worker-specific _representation_ of the project or event.

A more accurate description for this might be something along the lines of "encapsulates project-related details," since that hints that a "project" is a broader thing that exists elsewhere and this is a narrower view of it-- which is true.

Again, though. This is a nit. Dealer's choice on whether this is worth rewording or not.